scavenge/go.mod

```
@@ -1,19 +1,17 @@
-module github.com/cosmos/sdk-tutorials/scavenge
+module github.com/okwme/scavenge
```

```suggestion
module github.com/cosmos/sdk-tutorials/scavenge
```

will have to remove scavenge below

---

> will have to remove scavenge below

What do you mean by this actually?

---

remove from `go.mod`? (just did)